diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -209,15 +237,26 @@ function StudentQuiz(runtime, element, initData) {
       //he will be resumed to where he left.
 
       resizeContentContainer();
-      var completedStep = parseInt($(completedStepSelector, element).val());
-
-      //log event for xblock started
-      common.publishEvent({
+      var eventData = {
         event_type: 'xblock.diagnostic_feedback.quiz.started',
         quiz_type: initData.quiz_type,
-        quiz_title: initData.quiz_title,
-        current_question:  completedStep + 1
-      });
+        quiz_title: initData.quiz_title
+      };
+
+      var totalQuestions = $('.question-container').length;
```

Comment:
@jareerahsan To make sure that you're not running into problems if a unit contains more than one Diagnostic Feedback exercise, you should pass `element` as a second argument to jQuery: `$('.question-container', element)`. When you update the code it might be worth checking if you're doing this consistently throughout the JS code, it's easy to forget.